sign_oca/tests/test_sign_portal.py

```diff
@@ -71,3 +105,103 @@ def test_portal(self):
             val = self.request.signer_ids.get_info()["items"][key].copy()
             val["value"] = "My Name"
             data[key] = val
+
+    def test_get_sign_requests_domain(self):
+        self.domain = self.portal_sign.get_sign_requests_domain(request=self)
+        expected_domain = [
+            ("request_id.state", "=", "sent"),
+            ("partner_id", "child_of", [self.env.user.partner_id.id]),
+            ("signed_on", "=", False),
+        ]
+        self.assertEqual(sorted(self.domain), sorted(expected_domain))
```

Comment:
Personally I do not like this test, I think it would be more appropriate to make a portal test with a tour that checks that a user x when accessing the portal page can see the request previously created in setUp().

Example: https://github.com/OCA/rma/blob/b02f7ddceedca9ffc8b7e8ad44fe3cae315d82aa/rma_sale/tests/test_rma_sale_portal.py#L54